src/app/Scenes/MyCollection/Screens/Artwork/Components/MyCollectionArtworkHeader.tsx

```diff
@@ -29,6 +33,13 @@ export const MyCollectionArtworkHeader: React.FC<MyCollectionArtworkHeaderProps>
   const displaySubmissionStateSection =
     consignmentSubmission?.state && consignmentSubmission?.state !== "DRAFT"
 
+  const displaySubmissionStateSectionTier2 =
+    consignmentSubmission?.state && consignmentSubmission?.state !== "REJECTED" // TODO: Add more states to aviod displaying submissions in unsupprted statusess
+
+  const displaySubmissionStateSectionInHeader = enableSubmitArtworkTier2Information
+    ? displaySubmissionStateSectionTier2
+    : displaySubmissionStateSection
```

Comment:
Shouldn't this be equivalent? Because we always display the section when the artwork has been submitted.

```suggestion
  const displaySubmissionStateSection = enableSubmitArtworkTier2Information && consignmentSubmission?.state
```

Reply:
In case of a REJECTED state we should display the submission status under the demand index component, I think to not bring that much attention to the negative result. That's why I am checking for `consignmentSubmission?.state !== "REJECTED"`. But only in case if the ff is enabled

Reply:
Maybe checking for `consignmentSubmission?.state !== "REJECTED"` directly would be more explicit and easier to understand 🤔